### PR TITLE
added a question in linux-quiz.ts

### PR DESCRIPTION
**Pull request by shubhsinha**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by shubhsinha** [comment]
added a question for developer quiz site

File: src/data/linux-quiz.ts [hunk 1]
```diff
@@ -921,6 +921,16 @@ const linuxQuiz = [
       "The find command can be used to find files or folders matching a particular search pattern. It searches recursively.",
     Link: "https://www.freecodecamp.org/news/the-linux-commands-handbook/",
   },
+  {
+    Question: "Which of the following commands is used to calculate a file's total lines, words, and characters?",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:925]:
```suggestion
    Question: "Which of the following Linux commands calculates the total number of lines, words, and characters in a file?",
```

File: src/data/linux-quiz.ts [hunk 2]
```diff
@@ -921,6 +921,16 @@ const linuxQuiz = [
...
+    Answer: "wc",
+    Distractor1: "wordcount",
+    Distractor2: "wordc",
+    Distractor3: "lc",
+    Explanation:
+      "A file's total number of lines, words, and characters can be determined with the wc command.",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:931]:
```suggestion
      "The wc command calculates the total number of lines, words, and characters in a file.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by shubhsinha** [comment]
updated as requested

**Pull request merged**